exporter_test.go

```diff
@@ -102,6 +103,93 @@ func testExporter(t *testing.T, when spec.G, it spec.S) {
 	})
 
 	when("#Export", func() {
+		when("previous slice image exists", func() {
+			var (
+				fakeOriginalImage *fakes.Image
+				fakeImageMetadata lifecycle.LayersMetadata
+				sliceSHA          string
+			)
+
+			it.Before(func() {
+				var err error
+
+				appDir, err = filepath.Abs(filepath.Join("testdata", "exporter", "previous-slice-image-exists", "layers", "app"))
```

Comment:
Now that the `exporter` modifies the app dir (deletes files) we should work in a temp dir. Given that `app` is a subdir of the `layers`, and we recursively copy to a temp dir on line 126, we can just use that temp dir as `appDir` instead of the dir in the source tree.